libs/api/ebpf_api.cpp

```diff
@@ -2,55 +2,46 @@
 // SPDX-License-Identifier: MIT
 
 #include "pch.h"
-#include "ebpf_api.h"
 
-#include <map>
-#include <stdexcept>
 #include "api_common.hpp"
+#include "device_helper.hpp"
+#include "ebpf_api.h"
 #include "ebpf_protocol.h"
-#include "ebpf_platform.h"
 #include "map_descriptors.hpp"
-#include "platform.h"
+#include "rpc_client.h"
 extern "C"
 {
 #include "ubpf.h"
 }
 #include "Verifier.h"
-#include "verifier_service.h"
-#include "windows_helpers.hpp"
 
 #define MAX_CODE_SIZE (32 * 1024) // 32 KB
 
 uint32_t
 ebpf_api_initiate()
 {
-    if (device_handle != INVALID_HANDLE_VALUE) {
-        return ERROR_ALREADY_INITIALIZED;
-    }
+    uint32_t result;
 
-    device_handle = Platform::CreateFile(
-        EBPF_DEVICE_WIN32_NAME,
-        GENERIC_READ | GENERIC_WRITE,
-        0,
-        nullptr,
-        CREATE_ALWAYS,
-        FILE_ATTRIBUTE_NORMAL,
-        nullptr);
-
-    if (device_handle == INVALID_HANDLE_VALUE) {
-        return GetLastError();
+    result = initialize_device_handle();
```

Comment:
Logging?

Reply:
Yeah logging is missing from all the UM components. Filed #247 to add ETW tracing in all the UM components.